monitoring/error500.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable */
+
+const assert = require('assert');
+
+function backendCheck(alerts) {
+  const TIME_THRESHOLD = 60 * 1000 * 1000;
+  return alerts.sort((alert1, alert2) => {
+    if (alert1.time_start_usec < alert2.time_start_usec) {
+      return -1;
+    } else if (alert1.time_start_usec > alert2.time_start_usec) {
+      return 1;
+    } else {
+      return 0;
+    }
+  })
+    .filter((alert) => (alert.time_start_usec <= (parseInt(alerts[0].time_start_usec) + TIME_THRESHOLD).toString()));
+}
+
+function pagesMonitor(err, response, body) {
+  assert.equal(response.statusCode, 200, 'Expected a 200 OK response');
+  /* Get response and check for error clusters */
+  const ACTUAL_500 = [];
+  const EXPECTED = [];
+  const ACTUAL_50N = [];
+  const THRESHOLD = 3;
+
+  if (body.results != []) {
```

Comment:
[nitpicking] the `if` is not needed here.

Reply:
the test is wrong.

```
s = []
s != [] // alway true.
```

Reply:
the `if` is wrong and not needed :)